components/k8s-model-server/images/releaser/components/workflows.libsonnet

```diff
@@ -139,6 +224,100 @@
                 },
               ],
             },  // build-tf-serving-image
+            {
+              name: "deploy-tf-serving",
+              container: {
+                command: [
+                  "python",
+                  "-m",
+                  "testing.test_deploy",
+                  // "--project=mlkube-testing",
```

Comment:
I made you an owner of mlkube-testing so you can use that cluster if you want.
But if you want to override the cluster regardless, you should expose them as parameters so you can override the values for a specific ksonnet environment (e.g. create a lunkai-dev environment).

Reply:
thanks, done